pkg/client/logs.go

```diff
@@ -0,0 +1,35 @@
+package client
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// GetLogs fetches the logs from the provided Pod. If `numOfLines` is provided (instead of `nil`) then that
+// many lines will be returned from the end of the logs.
+func (c *Client) GetLogs(ctx context.Context, pod *corev1.Pod, numOfLines *int64) (string, error) {
+	coreClient, err := kubernetes.NewForConfig(c.config)
+	if err != nil {
+		return "", fmt.Errorf("failed initializing kubernetes core client - %v", err)
```

Comment:
my inner perfectionist wants Kubernetes to be capitalised, but that's just me being super picky :sweat_smile:

Reply:
I actually copied this same couple lines from another area of the code so don't want to have it inconsistent in this PR. might sort it everywhere later.

Reply:
that is an acceptable reason. Wealdy approves this message.